manifests/params.pp

```diff
@@ -46,6 +46,9 @@
   # stream settings
   $stream_port = 9102
 
+  # nginx basic status port
+  $basicstatus_port = 9103
```

Comment:
Is the new nginx basic status endpoint always on?

I think it is worth documenting and making it optional to make sure the configuration is consistent with the other deployments.